src/js/utils/dom.js

```diff
@@ -786,6 +786,10 @@ export function isSingleLeftClick(event) {
     return true;
   }
 
+  if (event.button === 0 && event.buttons === 0) {
```

Comment:
What does this condition check for?

Reply:
It checks if the left mouse button was pressed. According to the Mozzilla dev doc: 
```
event.button = 0 -> Main button pressed, usually the left button or the un-initialized state,
event.buttons  = 0 -> No button or un-initialized,
```
The problem: for 'mouseup' event we always get event.buttons equal to 0. (You can check here:
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons)
which means that we foll into the next if statement 
`if (event.button !== 0 || event.buttons !== 1) `
which always return false for the left mouse up click. So I added one extra 'if statement' to fix it but maybe there is a better way...  Do you have any thoughts about it?

Reply:
Seems reasonable, can you add a comment explaining we're account for mouseup and buttons being 0 in that case?

Reply:
Yes, sure.